subscriptionform/templates/registration/login.html

```diff
@@ -1,37 +1,38 @@
 {% extends "base.html" %}
+{% load i18n %}
 
 {% block content %}
 
 {% if form.errors %}
-<p>Your username and password didn't match. Please try again.</p>
+    <p>{% trans "Your username and password didn't match. Please try again." %}</p>
 {% endif %}
 
 {% if next %}
     {% if user.is_authenticated %}
-    <p>Your account doesn't have access to this page. To proceed,
-    please login with an account that has access.</p>
+        <p>{% trans "Your account doesn't have access to this page. To proceed,
+        please login with an account that has access." %}</p>
     {% else %}
-    <p>Please login to see this page.</p>
+        <p>{% trans 'Please login to see this page.' %}</p>
```

Comment:
graag consequent zijn in gebruik van type quotes. Kijk ook eens naar blocktrans voor grote stukken tekst. Dit is een randgevalletje